docs/learning/ECSqlReference/CASE.md

```diff
@@ -0,0 +1,50 @@
+# CASE-WHEN-THEN-ELSE
+
+ECSQL supports only searched CASE expressions:
```

Comment:
Not sure I understand. Might need re-wording?

Reply:
This is actually what this flavor of CASE statement is called: https://learn.microsoft.com/en-us/sql/t-sql/language-elements/case-transact-sql?view=sql-server-ver16#syntax

Reply:
Thanks